sdk/servicebus/service-bus/samples/typescript/gettingStarted/atomManagementApi.ts

```diff
@@ -0,0 +1,56 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+ Below are the management operations that you can do with queues. Simiilar support exists
```

Comment:
Following the pattern in other samples in this project... 
```suggestion
 This sample demonstrates how to carry out management operations using connection string as the means of authenticating to Azure Service Bus. While this sample shows queue related management operations, you can do the same for 
```